app/src/main/java/com/example/armando/project/logIn.java

```diff
@@ -0,0 +1,11 @@
+package com.example.armando.project;
+
+public class logIn {
+
+    public static int validate(String inputUser, String inputPass) {
+        if (inputUser.equals("user") && inputPass.equals("pass"))
+            return 1;
+        else
+            return 0;
+    }
```

Comment:
Is there a reason that we use int's instead of boolean's here?

Reply:
Right, this was for an error check when I had Firebase references in this class. It'll work just fine, and make more sense if I change it to Booleans.